AcloConcurrency/AcloConcurrency/MyPlaces/ViewModel/PlacesViewModel.swift

```diff
@@ -0,0 +1,24 @@
+import Combine
+
+public class PlacesViewModel: ObservableObject {
+    
+    @Published private(set) var places: [PlaceModel] = []
+    
+    private let userStorage: UserStorage
+    
+    public init(userStorage: UserStorage) {
+        self.userStorage = userStorage
+    }
+    @MainActor
+    func getPlaces() {
+        Task{
+            self.places = userStorage.places
+        }
+    }
+    @MainActor func savePlace(_ place: PlaceModel) {
+        Task{
+            userStorage.saveNewPlace(place)
+        }
+        getPlaces()
+    }
```

Comment:
Right now we don't need here MainActor and Task. You can remove and later we think about how to use async